vtkext/private/module/vtkF3DEXRReader.cxx

```diff
@@ -142,6 +223,24 @@ void vtkF3DEXRReader::ExecuteDataWithInformation(vtkDataObject* output, vtkInfor
   }
 }
 
+//------------------------------------------------------------------------------
+/**
+ * Read from memory instead of file
+ */
+void vtkF3DEXRReader::SetMemoryBuffer(const void* buff)
+{
+  MemoryBuffer = buff;
```

Comment:
this->MemoryBuffer

Reply:
this is the mother class implementation, why not just rely on it instead of defining your own  ?

```
void vtkImageReader2::SetMemoryBuffer(const void* membuf)
{
  if (this->MemoryBuffer != membuf)
  {
    this->MemoryBuffer = membuf;
    this->Modified();
  }
}

//------------------------------------------------------------------------------
void vtkImageReader2::SetMemoryBufferLength(vtkIdType buflen)
{
  if (this->MemoryBufferLength != buflen)
  {
    this->MemoryBufferLength = buflen;
    this->Modified();
  }
}
```

Reply:
I used this for debugging, it can now be removed.